### PR TITLE
Fix mod reloading for MAD

### PR DESCRIPTION
**Pull request by RoyalBlue1**
Game does not load forever after downloading mod though auto download anymore

### CODE REVIEW

**Review by ASpoonPlaysGames — APPROVED** [review]
Code looks aight i guess :/

Also seemed to work in testing

File: Northstar.Client/mod/scripts/vscripts/ui/menu_ns_connect_password.nut [hunk 1]
```diff
@@ -56,6 +56,6 @@ void function ConnectWithPassword( var button )
 	if ( GetTopNonDialogMenu() == file.menu )
 	{
 		TriggerConnectToServerCallbacks()
-		thread ThreadedAuthAndConnectToServer( Hud_GetUTF8Text( Hud_GetChild( file.menu, "EnterPasswordBox" ) ) )
+		thread ThreadedAuthAndConnectToServer( Hud_GetUTF8Text( Hud_GetChild( file.menu, "EnterPasswordBox" ) ), true )
```
Comment (ASpoonPlaysGames) [inline Northstar.Client/mod/scripts/vscripts/ui/menu_ns_connect_password.nut:59]: This should be changed later, moving mod downloading to after password and such

**Review by GeckoEidechse — APPROVED** [review]
Confirmed working in testing. Using a fresh profile with the PR and it loaded just fine :D

File: Northstar.Client/mod/scripts/vscripts/ui/menu_ns_connect_password.nut (hunk 1 above)
Comment (GeckoEidechse) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_ns_connect_password.nut:59]: Moved to #781

**Pull request merged**